### PR TITLE
Added Hussainstest.java

### PR DESCRIPTION
**Pull request by HussainArif12**

### CODE REVIEW

**Comment by HussainArif12** [comment]
Added file

**Comment by HussainArif12** [comment]
Sir I'm having build errors.what do i do?

**Comment by vicens-aniol** [comment]
solve it

**Comment by HussainArif12** [comment]
It can't! The Travis CI log says
Assert.assertTrue(..)
Could not find symbol : Assert

**Comment by SachinRammoorthy** [comment]
@HussainArif12 you have to import `Assert` through `import org.junit.Assert;` . Also, why do you have 2 test files? This also causes an error as they have the same name.

**Comment by shubhamvashisht** [comment]
@HussainArif12 Why did you edit travis file?
And why did you write tests in multiple files?

File: src/test/java/org/jbossoutreach/HussainsTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import org.junit.Test;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+import org.junit.runner.notification.Failure;
+
```
Comment (shubhamvashisht) [inline src/test/java/org/jbossoutreach/HussainsTest.java]: Travis is showing error for this file because you haven't imported Assert class.

**Pull request closed without merging**